cfgmgr/portmgr.cpp

```diff
@@ -256,4 +319,4 @@ bool PortMgr::writeConfigToAppDb(const std::string &alias, std::vector<FieldValu
 {
     m_appPortTable.set(alias, field_values);
     return true;
-}
```

Comment:
What changed here? Please remove unintended changes wrt spacing and new lines

Reply:
removed white spaces and new lines